### PR TITLE
Docs: minor changes and typos

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

File: docs_src/manuals/getting_started/getting_started.rst [hunk 1]
```diff
@@ -114,16 +116,16 @@ For efficient memory accesses the index ordering might depend on the target arch
...
+the setter methods and finally call the ``build()`` method which returns a ``std::shared_ptr``
+of a newly created storage. The builder also provides overloaded call operator which is a synonym of ``build()``.
 There are two required properties that need to be set:
 the type of element, eg ``.type<double>()``, and the sizes for each dimension, eg ``.dimensions(10, 12, 20)``.
 Other properties are optional.
 
-If we need several storages that share some properties, we can construct partially specified builder with
-the shared properties set and to reuse it while building concrete storages.
+If we need several storages that share some properties, we can construct a partially specified builder with
+the set of shared properties and reuse it while building concrete storages.
```
[the remark this answers is not in the document]
Comment (havogt) [reply docs_src/manuals/getting_started/getting_started.rst:128]: Thanks! I didn't understand that it was a verb in the first sentence. Rewrote again.

File: docs/_sources/introduction/introduction.rst.txt [hunk 2]
```diff
@@ -50,160 +50,177 @@ computations in order to be optimal on the target hardware.
...
+ target_link_libraries(my_library PUBLIC GridTools::stencil_cuda)
+ gridtools_setup_target(my_library CUDA_ARCH sm_60)
+
+
+^^^^^^^^^^^^^^^
+AMD GPU Support
+^^^^^^^^^^^^^^^
+
+Further, GridTools can also be compiled for AMD GPUs using AMD’s HIP. But due to limitations of AMD’s current HCC compiler,
+the Clang-based HIP version has to be used, which is not yet officially distributed in the current HIP releases
```
[the remark this answers is not in the document]
Comment (havogt) [reply docs/_sources/introduction/introduction.rst.txt:214]: improved

[the remark this answers is not in the document]
Comment (fthaler) [reply docs/_sources/introduction/introduction.rst.txt:214]: Maybe we can also just state, that we require ROCm 3.5 or newer due to the compiler requirement? If this is met, the user does not have to compile HIP-Clang, as it is already included.

File: docs_src/manuals/introduction/introduction.rst [hunk 3]
```diff
@@ -210,9 +210,8 @@ allows to compile the same *.cpp* file for both CUDA and host, without having to
 AMD GPU Support
 ^^^^^^^^^^^^^^^
 
-Further, GridTools can also be compiled for AMD GPUs using AMD’s HIP. But due to limitations of AMD’s current HCC compiler,
-the Clang-based HIP version has to be used, which is not yet officially distributed in the current HIP releases
-(but will be the successor of HCC-based HIP). Compilation of a Clang-based HIP compiler is relatively straightforward and
+Further, |GT| can also be compiled for AMD GPUs using AMD’s HIP. To compile |GT| you need the Clang-based HIP copmpiler.
+Compilation of a Clang-based HIP compiler is relatively straightforward and
 documented in the `official HIP repository <https://github.com/ROCm-Developer-Tools/HIP/blob/master/INSTALL.md#hip-clang>`_.
```
Comment (havogt) [inline docs_src/manuals/introduction/introduction.rst:215]:
```suggestion
Further, |GT| can also be compiled for AMD GPUs using AMD’s HIP. To compile |GT| you need the Clang-based HIP compiler which is available with ROCm 3.5 and later.
```

Comment (havogt) [reply docs_src/manuals/introduction/introduction.rst:215]: Something like this?

File: docs/_sources/introduction/introduction.rst.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (havogt) [reply docs/_sources/introduction/introduction.rst.txt:214]: See below.

File: docs_src/manuals/introduction/introduction.rst (hunk 3 above)
Comment (fthaler) [reply docs_src/manuals/introduction/introduction.rst]: :+1:

**Pull request merged**